Gemfile

```diff
@@ -6,6 +6,8 @@ gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
 
 group :development do
   gem 'rspec'
+  gem 'coveralls'
+  gem 'simplecov'  
```

Comment:
https://github.com/dmcnulla/ntp/blob/coverage/features/step_definitions/server_steps.rb 
Lines 17 & 28 use expect(). This is an rspec method.
...
OK. I replaced rspec with rspec-expecations. The test failed because it required rspec/mock. So I reverted that change. I think rspec is needed.